lib/qs/qs_runner.rb

```diff
@@ -17,6 +32,18 @@ def run_callbacks(callbacks)
       callbacks.each{ |proc| self.handler.instance_eval(&proc) }
     end
 
+    module OptionalTimeout
+      def self.after(timeout, &block)
```

Comment:
@jcredding saw the stubbing below - having the module function makes stubbing nicer.